app/questionnaire/placeholder_renderer.py

```diff
@@ -39,13 +39,17 @@ def __init__(
 
     def render_pointer(
         self,
+        *,
         dict_to_render: Mapping[str, Any],
         pointer_to_render: str,
         list_item_id: Optional[str],
+        routing_path_block_ids: Optional[tuple] = None,
```

Comment:
This shouldn't be passed to pointer, this should be on the object itself.

Reply:
Moved to the object now 👍